File: Java/TP3-4/src/Usuario.java
import java.util.ArrayList;

public class Usuario {
	//Armazenador de Strings...
	private ArrayList<String> login;

	public Usuario(String login) {
		this.login = new ArrayList<>();
		this.login.add(login);
	}

	public ArrayList<String> getLogin() {
		return login;
	}

	public boolean buscaLogin(String login) {
		return this.login.contains(login);
	}

	public void Saudacao() {
		System.out.println("Bem-vindo " + login + ".");
	}
}

File: Kotlin/TPK4-4/bin/Usuario.kt
import java.util.ArrayList
import java.util.InputMismatchException

class Usuario @Throws(InputMismatchException::class)
constructor(login: String) {
	//Armazenador de Strings...
	private var login: ArrayList<String>

	init {
		this.login = ArrayList<String>()
		this.login.add(login)
	}

	fun buscaLogin(login: String): Boolean {
		return this.login.contains(login)
	}

	fun Saudacao() {
		println("Bem-vindo " + login + ".")
	}

	fun descricaoUsuario() {
		println("Log-off " + login)
	}

	fun despedidaUsuario() {
		println("")
	}

	fun getLogin(): ArrayList<String> {
		return login;
	}
}